go/vt/discovery/keyspace_events.go

```diff
@@ -221,7 +221,7 @@ func (kew *KeyspaceEventWatcher) broadcast(ev *KeyspaceEvent) {
 }
 
 func (kew *KeyspaceEventWatcher) run(ctx context.Context) {
-	hcChan := kew.hc.Subscribe()
+	hcChan := kew.hc.Subscribe("KeyspaceEventWatcher")
```

Comment:
nit: I feel like all these subscriber names should be constants.

Reply:
Done